<?xml version="1.0"?>
<launch>
    <!-- roomba number -->
    <arg name="roomba" default="roomba3"/>

    <!-- topic name -->
    <arg name="image_topic" default="/$(arg roomba)/camera/color/image_rect_color"/>
    <arg name="bbox_topic" default="/$(arg roomba)/bounding_boxes"/>

    <!-- launch node -->
    <node pkg="collaborative_vslam" type="roomba_mask_creator_node" name="roomba_mask_image" output="screen">
        <!-- topic name -->
        <remap from="/image_raw" to="$(arg image_topic)"/>
        <remap from="/bboxes" to="$(arg bbox_topic)"/>
        <remap from="/mask_image" to="/$(arg roomba)/mask_image"/>

        <!-- parameter -->
        <param name="/hz" value="15"/>
    </node>
</launch>
